packages/runtime/container-runtime/src/summaryCollection.ts

```diff
@@ -298,7 +298,18 @@ export class SummaryCollection {
         const seq = op.contents.summaryProposal.summarySequenceNumber;
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         const summary = this.pendingSummaries.get(seq)!;
-        assert(!!summary); // We should never see an ack without an op
+        if (!summary) {
+            // Summary ack without an op should be rare. We could fetch the
```

Comment:
Given it's rare event, can we have here an error telemetry event? It's no different than us recorded 404, which happens when user deleted a file. Even better is to add "warning" category, probably.